privacy_evaluator/models/torch/cnn.py

```diff
@@ -23,56 +29,67 @@ def ConvNet(
         )
 
 
+class ConvBlock(nn.Module):
+    def __init__(self, in_channels, out_channels):
```

Comment:
Copy pasted from pull request #160 to avoid this comment from getting lost when #160 is deleted: 
yzchyx said: maybe we can specify the type for in_channels and out_channels, from the conv-layer we could say the two parameters are two ints instead of two lists of int, right?

Reply:
Of course, we can specify that these two parameters are ints, but they are not defined as two lists of type int. The class expects already to get these two parameters as ints.

Reply:
The types were added